_typography.scss

```diff
@@ -18,6 +18,10 @@ $gel-settings-available: false !default;
 $core: true !default;
 $enhanced: true !default;
 
+// If larger font sizes enable, merge them with gel-type-settings
+@if $gel-type-enable--larger-font-sizes {
```

Comment:
The style seems to be using the words type or typography. Could we change references to "font" to fit in with this?

Reply:
Thanks. Will update to be consistent with naming.